### ISSUE TITLE
Compilation error when use -DETL_BLAS_MODE 

### ISSUE THREAD

**Issue by qipx** [wichtounet/dll#11]
Hello.
When I use -DETL_BLAS_MODE flag to compile, I get these error. I have already installed liblapack.
```
clang++-4.0 -DETL_PARALLEL -DETL_VECTORIZE_FUL -DETL_BLAS_MODE -std=c++1z -lpthread -llapack -lblas  main.cpp
In file included from main.cpp:8:
In file included from dll/neural/conv_layer.hpp:11:
In file included from dll/neural/dyn_conv_layer.hpp:10:
In file included from dll/neural/dyn_conv_layer_impl.hpp:11:
In file included from dll/neural_layer.hpp:15:
In file included from dll/etl/etl.hpp:108:
In file included from dll/etl/expr/gemm_expr.hpp:15:
dll/etl/impl/blas/gemm.hpp:952:32: error: use of undeclared identifier 'mkl_get_max_threads'
            auto mkl_threads = mkl_get_max_threads();
                               ^
dll/etl/impl/blas/gemm.hpp:954:13: error: use of undeclared identifier 'mkl_set_num_threads'
            mkl_set_num_threads(1);
            ^
dll/etl/impl/blas/gemm.hpp:1250:32: error: use of undeclared identifier 'mkl_get_max_threads'
            auto mkl_threads = mkl_get_max_threads();
                               ^
dll/etl/impl/blas/gemm.hpp:1252:13: error: use of undeclared identifier 'mkl_set_num_threads'
            mkl_set_num_threads(1);
            ^
4 errors generated.

```
any solutions?

**Comment by wichtounet** [comment]
Hello qipx, 

This is a bug of mine. I will fix it tomorrow. It should be an easy fix. 

Sorry about that

**Comment by qipx** [comment]
Thanks.
I commented out error lines and the program works well.
This library is very cool.

**Comment by wichtounet** [comment]
This should be fixed with 1f451ea

I'm glad you could fix it yourself. And glad you like :)

Tell me if you still have issues.
